docs/src/data/frameworks.ts

```diff
@@ -27,9 +27,9 @@ export const REACT_NATIVE_DEPENDENCIES =
   'react-native-safe-area-context amazon-cognito-identity-js @react-native-community/netinfo @react-native-async-storage/async-storage react-native-get-random-values react-native-url-polyfill';
 
 export const FRAMEWORK_INSTALL_SCRIPTS = {
-  react: '@aws-amplify/ui-react aws-amplify',
-  vue: '@aws-amplify/ui-vue aws-amplify',
-  angular: '@aws-amplify/ui-angular aws-amplify',
+  react: 'aws-amplify @aws-amplify/ui-react',
```

Comment:
would prefer to keep aws-amplify on the end, since we are the ui library.

Reply:
I think we should put `aws-amplify` first because we want to install packages from big to small.

1) other docs site usually put more basic package first, 
2) `aws-amplify` is the prerequisite of `@aws-amplify/ui-x`
3) the end of the line is more visible than the middle

Reply:
> aws-amplify is the prerequisite of @aws-amplify/ui-x

`aws-amplify` is a peer dep of our packages, and ui-react is usable without it

> the end of the line is more visible than the middle

For React Native, placing `aws-amplify` first leaves `@aws-amplify/ui-react-native` in the middle of the install command due to the extended deps required

Reply:
Respectfully disagree, I don't think there's a compelling reason to change it. I haven't heard of any standard about installing big to small package order. I think the ui package makes the most sense on the front due to being on the UI Docs site, and because the primary dependency customers are interested on this site is the ui package.

Example:
https://chakra-ui.com/getting-started
<img width="806" alt="image" src="https://user-images.githubusercontent.com/6165315/214649912-e8bf4aa3-f051-47a8-9c85-289506bb7566.png">

Reply:
Ok. Reset this part